src/css/molecules/header.scss

```
@@ -51,3 +51,22 @@
     }
   }
 }
+
+.mzp-c-menu-category-list .site-search-form {
+  margin: 4px 0px 0px 0px;
```

Do not use units with `0` values.

```suggestion
  margin: 4px 0 0;
```